foreshadow/transformers/transformers.py

```diff
@@ -22,32 +40,48 @@ def _get_modules(classes, _globals, mname):
 
     Args:
         classes: A list of classes
-        _globals: The globals in the callee's context
+        globals_: The globals in the callee's context
         mname: The module name
+        wrap: True to wrap modules, False to not (manual decorator
+            make_panads_transformer) will need to be applied.
 
     Returns:
         The list of wrapped transformers.
 
     """
-    transformers = [
-        cls
-        for cls in classes
-        if issubclass(cls, BaseEstimator)
-        and (
-            issubclass(cls, TransformerMixin)
-            or issubclass(cls, VectorizerMixin)
-        )
-    ]
+
+    def no_wrap(f):
+        """Don't wrap function f. Return unwrapped function pointer.
+
+        Args:
+            f: function point f.
+
+        Returns:
+            function pointer, f
+
+        """
+        return f
+
+    transformers = []
```

Comment:
might make sense to turn this into a list comprehension for a speed boost.

Reply:
will do